parsl/tests/configs/slurm_local.py

```diff
@@ -0,0 +1,29 @@
+from parsl.channels import LocalChannel
+from parsl.config import Config
+from parsl.executors import HighThroughputExecutor
+from parsl.launchers import SrunLauncher
+from parsl.providers import SlurmProvider
+
+
+def fresh_config():
+    return Config(
+        executors=[
+            HighThroughputExecutor(
+                label="docker_slurm",
+                encrypted=True,
+                provider=SlurmProvider(
+                    cmd_timeout=60,     # Add extra time for slow scheduler responses
+                    channel=LocalChannel(),
+                    nodes_per_block=1,
+                    init_blocks=1,
+                    min_blocks=1,
+                    max_blocks=1,
+                    walltime='00:10:00',
+                    launcher=SrunLauncher(),
+                ),
+            )
+        ],
+    )
+
+
+config = fresh_config()
```

Comment:
I think this line isn't needed and you got it by cut and pasting it from somewhere else that also doesn't need it... it also causes trouble in some situations by making configs be initialized at import (so every single test config gets initialised on every test run), rather than when the config is actually wanted.